### ISSUE TITLE
Overwrite must be forbidden on public links with drop zone

### ISSUE THREAD

**Issue by labkode** [cs3org/reva#2480]
If two people upload the same file they will overwrite each other.
The expected behaviour is to to make each upload unique.
In CERNBox we prepend a random UUID.

<img width="1440" alt="Screenshot 2022-01-27 at 14 35 18" src="https://user-images.githubusercontent.com/5175912/151369707-e8fd5304-c5d5-4d20-89c1-01fa61317835.png">

**Comment by pmaier1** [comment]
I'd like to advocate against using UUIDs here as they appear very technical and regular users might feel uncomfortable. Instead I'd propose to *append* a number in parentheses "`(2)`". This is the behavior we have in the classic ownCloud. /cc @tbsbdr

**Comment by ishank011** [comment]
I think appending the timestamp might be even better? That'll help the user differentiate between the various files

**Comment by pmaier1** [comment]
> I think appending the timestamp might be even better? That'll help the user differentiate between the various files

Agree, that could be valuable as well. Dropbox behaves like this, for example.

**Comment by labkode** [comment]
@pmaier1 make sense!

**Comment by phil-davis** [comment]
What is happening here? We are happy to adjust tests. There has been plenty of discussion in PR #2510 - we just need a decision on the naming format to move forward.
